CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+- Add: Make initial notification in ONCHANGE notifications configurable (#920)
```

Comment:
ONCHANGE is an old terminology, no longer used in Orion. I'd suggest to change to:

```
- Add: skipInitialNotification URI param to make initial notification in subscriptions configurable (#920)
```

Reply:
Fixed in b0f68c8